sharding/oneShardCoordinator_test.go

```diff
@@ -0,0 +1,56 @@
+package sharding
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOneShardCoordinator_NumberOfShardsShouldWork(t *testing.T) {
```

Comment:
since `OneShardCoordinator` is very simple, we can have here only one bigger test to include all functions;
so one instantiation for `oneShardCoordinator`

Reply:
updated

Reply:
we can remove the inner tests here, and just getter functions from the main test

Reply:
solved